src/main/java/uk/nhs/digital/nhsconnect/nhais/model/edifact/AcceptanceType.java

```diff
@@ -0,0 +1,35 @@
+package uk.nhs.digital.nhsconnect.nhais.model.edifact;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.NonNull;
+import uk.nhs.digital.nhsconnect.nhais.exceptions.EdifactValidationException;
+
+@Builder
+@Data
+public class AcceptanceType extends Segment {
+    private final static String APT_PREFIX = "ATP";
+    private final static String ZZZ_SUFFIX = ":ZZZ";
+    private @NonNull String type;
```

Comment:
This is always a numerical code, do we want an enum to map between them?

Reply:
I'm using map in AcceptanceTypeMapper:
`    private final static Map<String, String> ACC_TYPE_MAPPING = ImmutableMap.of(
            "birth", "1",
            "first", "2",
            "transferin", "3",
            "immigrant", "4",
            "exservices", "5"
    );`

Reply:
validation added to AcceptanceType.java